odl/set/domain.py

```diff
@@ -241,13 +246,79 @@ def contains_set(self, other, tol=0.0):
             The maximum allowed distance in 'inf'-norm between the
             other set and this interval product.
             Default: 0.0
+
+        Examples
+        --------
+        >>> b1, e1 = [-1, 0, 2], [-0.5, 0, 3]
+        >>> rbox1 = IntervalProd(b1, e1)
+        >>> b2, e2 = [-0.6, 0, 2.1], [-0.5, 0, 2.5]
+        >>> rbox2 = IntervalProd(b2, e2)
+        >>> rbox1.contains_set(rbox2)
+        True
+        >>> rbox2.contains_set(rbox1)
+        False
         """
         try:
             return (self.approx_contains(other.min(), tol) and
                     self.approx_contains(other.max(), tol))
-        except AttributeError:
-            raise AttributeError('cannot test {!r} without `min()` and `max()`'
-                                 'methods.'.format(other))
+        except AttributeError as exc:
+            raise_from(
+                AttributeError('cannot test {!r} without `min()` and `max()`'
+                               'methods.'.format(other)), exc)
+
+    def contains_all(self, other):
+        """Test if all points defined by `other` are contained.
+
+        Parameters
+        ----------
+        other : object
+            Can be a single point, a ``(d, N)`` array where ``d`` is the
+            number of dimensions or a length-`d` meshgrid sequence
+
+        Returns
+        -------
+        contains : bool
+            `True` if all points are contained, `False` otherwise
+
+        Examples
+        --------
+        >>> b, e = [-1, 0, 2], [-0.5, 0, 3]
+        >>> rbox = IntervalProd(b, e)
+        ...
+        ... # Arrays are expected in (ndim, npoints) shape
+        >>> arr = np.array([[-1, 0, 2],   # defining one point at a time
+        ...                 [-0.5, 0, 2],
+        ...                 [-0.5, 0, 2.8],
+        ...                 [-0.75, 0, 3]])
+        >>> rbox.contains_all(arr.T)
+        True
+        >>> # Implicit meshgrids defined by coordinate vectors
+        >>> vec1 = (-1, -0.9, -0.7)
+        >>> vec2 = (0, 0, 0)
+        >>> vec3 = (2.5, 2.75, 3)
+        >>> mg = np.meshgrid(vec1, vec2, vec3,
+        ...                  sparse=True, indexing='ij')
```

Comment:
Otherwise axes are swapped in the resulting arrays. An alternative would be to use our own `sparse_meshgrid` function.